test/with-typescript/index.ts

```diff
@@ -1689,7 +1670,10 @@ async function getSessionWithoutRequestWithErrorHandler(req: express.Request, re
 async function createNewSessionWithoutRequestResponse(req: express.Request, resp: express.Response) {
     const userId = "user-id"; // This would be fetched from somewhere
 
-    const session = await Session.createNewSessionWithoutRequestResponse("public", userId);
+    const session = await Session.createNewSessionWithoutRequestResponse(
+        "public",
+        Supertokens.convertToRecipeUserId(userId)
+    );
 
```

Comment:
- modify this to use new functions as well
- add the type of User for some variable (we need to make sure that the dev can import the user type)
- add the type of recipe user if to some variable (we need to make sure that the dev can import recipeuserid type)